neo4j/v1/bolt.py

```diff
@@ -238,19 +243,21 @@ def __init__(self, sock, **config):
         # Pick up the server certificate, if any
         self.der_encoded_server_certificate = config.get("der_encoded_server_certificate")
 
+        def on_success(metadata):
+            self.server_version = metadata.get("server")
```

Comment:
Just used in a test right now. It will conflict with #108 so I'll adjust it properly for that.